src/components/Extension/PrivateExtensions/actions/IntakeScheduling/api.service.ts

```diff
@@ -93,7 +102,7 @@ export const fetchAvailability = async ({
 }): Promise<GetAvailabilitiesResponseType> => {
   try {
     const response = await fetch(
-      `/api/sol/providers/${input.providerId[0]}/availability`,
+      `/api/sol/providers/${input.providerId[0]}/availability?session_id=${requestOptions.logContext?.session?.id}&pathway_id=${requestOptions.logContext?.session?.pathway_id}`,
```

Comment:
I'm not a huge fan of letting know the user we are passing ids in the URL - could we rename it to `session=xxx&careflow=xxx` (we are trying to move away from pathway naming convention)